src/gui/translations.cpp

```diff
@@ -42,20 +42,11 @@ namespace Translations {
         return QStringLiteral(".qm");
     }
 
-    QString applicationTrPath()
+    QDir translationsDirectory()
     {
-        const auto devTrPath = QDir(qApp->applicationDirPath() + QStringLiteral("/../src/gui/"));
-        if (devTrPath.exists()) {
-            // might miss Qt, QtKeyChain, etc.
-            qCWarning(lcTranslations) << "Running from build location! Translations may be incomplete!";
-            return devTrPath.absolutePath();
-        }
-#ifdef Q_OS_MAC
-        const auto translationDir = QStringLiteral("Translations");
-#else
-        const auto translationDir = QStringLiteral("i18n");
-#endif
-        return QStandardPaths::locate(QStandardPaths::AppDataLocation, translationDir, QStandardPaths::LocateDirectory);
+        QDir dir(":/client/" APPLICATION_SHORTNAME "/translations/");
+        Q_ASSERT(dir.exists());
```

Comment:
Translations are optional and at the moment the squish tests are not building them
```09:07:21:300 AUT stdout: 07-27 09:07:20:390 [ fatal default ]:	ASSERT: "dir.exists()" in file ../src/gui/translations.cpp, line 48